Qapple/Qapple/SourceCode/Feature/1.MainFlow/MainFlowView.swift

```diff
@@ -12,29 +12,37 @@ struct MainFlowView: View {
     
     @Bindable var store: StoreOf<MainFlowFeature>
     
+    @State private var tab: MainFlowFeature.MainFlowTab = .question
+    
     var body: some View {
         NavigationStack(path: $store.scope(state: \.path, action: \.path)) {
-            TabView {
+            TabView(selection: $tab) {
                 QuestionTabView(store: store.scope(state: \.questionTab, action: \.questionTab))
+                    .tag(MainFlowFeature.MainFlowTab.question)
                     .tabItem {
                         Image(systemName: "questionmark.bubble.fill")
                         Text("오늘의 질문")
                     }
                 
                 BulletinBoardView(store: store.scope(state: \.bulletinBoardTab, action: \.bulletinBoardTab))
+                    .tag(MainFlowFeature.MainFlowTab.bulletinBoard)
                     .tabItem {
                         Image(systemName: "list.clipboard.fill")
                         Text("게시판")
                     }
                 
                 ProfileView(store: store.scope(state: \.profileTab, action: \.profileTab))
+                    .tag(MainFlowFeature.MainFlowTab.profile)
                     .tabItem {
                         Image(systemName: "person.fill")
                         Text("내 정보")
                     }
             }
             .tint(.button)
             .fixedTabBarBackground(color: .first)
+            .onChange(of: tab) { _, _ in
+                HapticService.impact(style: .soft)
+            }
```

Comment:
탭바가 전환될 때 햅틱 이벤트를 추가했습니다.

Reply:
onChange 메소드에 파라미터 안써도 되서 지워도 좋을 듯 합니다!
```suggestion
            .onChange(of: tab) {
                HapticService.impact(style: .soft)
            }
```

Reply:
오 안써줘도 되는지 처음 알았네요! 감삼당